podcasts/TranscriptsViewController.swift

```diff
@@ -34,12 +34,12 @@ class TranscriptsViewController: PlayerItemViewController {
             [
                 transcriptView.topAnchor.constraint(equalTo: view.topAnchor),
                 transcriptView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
-                transcriptView.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 32),
-                transcriptView.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -32)
+                transcriptView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+                transcriptView.trailingAnchor.constraint(equalTo: view.trailingAnchor)
             ]
         )
 
-        transcriptView.textContainerInset = .init(top: 0.75 * Sizes.topGradientHeight, left: 0, bottom: 0.7 * Sizes.bottomGradientHeight, right: 0)
+        transcriptView.textContainerInset = .init(top: 0.75 * Sizes.topGradientHeight, left: 32, bottom: 0.7 * Sizes.bottomGradientHeight, right: 32)
```

Comment:
We should use [readableContentGuide](https://developer.apple.com/documentation/uikit/uiview/1622644-readablecontentguide) for iPad to prevent the text from filling the whole screen.

Reply:
Good suggestion. Done here: https://github.com/Automattic/pocket-casts-ios/pull/1973/commits/8141ff73d7dea2ca4b8f8aaf8e8527dd473daf1b

Just a note that on the implementation I did not use directly the layout guide anchors because I wanted to keep the scroll insets on the edges of the device.